cli/lib/nf3.mjs

```diff
@@ -222,14 +224,20 @@ class Nf3 {
   async estimateGas(contractAddress, unsignedTransaction) {
     let gasLimit;
     try {
-      // eslint-disable-next-line no-await-in-loop
-      gasLimit = await this.web3.eth.estimateGas({
-        from: this.ethereumAddress,
-        to: contractAddress,
-        data: unsignedTransaction,
+      const res = await axios.post(this.web3WsUrl, {
```

Comment:
Can you add some comments as to why you’ve done this? I want to prevent somebody reverting this in the future